src/api/operations/household.fragments.graphql

```diff
@@ -35,6 +26,22 @@ fragment HouseholdClientFields on HouseholdClient {
   }
 }
 
+fragment HouseholdClientFieldsClient on Client {
```

Comment:
This fragment name is extremely awkward-- I'm trying to reflect that it's the `Client` fragment internal to the `HouseholdClientFields` fragment. Is this ok or does this name spaghetti indicate too much code spaghetti?